src/gui.h

```diff
@@ -167,7 +167,7 @@ extern char window_created;
 
 void destroy_window(void);
 void create_gc(void);
-void set_struts(int);
+void set_struts(alignment);
```

Comment:
- [ ] I kept this line like this for a reason, it previously caused compilation to fail because `alignment` wasn't available to some file (wayland?) that uses it. Note for self to check before merge.